openshift/templates/global.config.yaml

```diff
@@ -0,0 +1,25 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: wps-api-global
+objects:
+- kind: ConfigMap
+  apiVersion: v1
+  metadata:
+    name: wps-api-global
+    labels:
+      app.kubernetes.io/part-of: wps-api
+  data:
+    env.origins: wps-web-*.pathfinder.gov.bc.ca
+    env.spotwx-url: 'https://spotwx.com/services/api.php'
+    env.wfwx-auth-url:
+    env.wfwx-base-url:
+    env.wfwx-user:
+- kind: Secret
+  apiVersion: v1
+  metadata:
+    name: wps-api-global
+  type: Opaque
+  data:
+    spotwx-api-key:
+    wfwx-secret:
```

Comment:
```suggestion
    wfwx-secret:
  parameters:
  - description: Wildfire One API authentication URL
  - name: WFWX-AUTH-URL
```

Reply:
Can you write `parameters` field right before `objects` field for consistency?